internal/pkg/template/templates/environment/partials/custom-resources.yml

```diff
@@ -45,7 +45,7 @@ HTTPSCert:
 CustomDomainAction:
   Metadata:
     'aws:copilot:description': 'Add an A-record to the hosted zone for the domain alias'
-  Condition: ManagedAliases
+  Condition: CreateCerts
```

Comment:
ah, this action is used by both alb and cdn. Which means that if all service within an env imports alb, then the cdn won't get any of the aliases. Probably not an issue though - because cdn needs to route to the managed alb anyway.

I think the implication here is that, when we talk about `ManagedAliases`, we are really talking about aliases that are:
1. Managed by the app-level domain
2. Will route to our managed alb (not the imported)

Our existing `ManagedAliases` condition only reflects the first statement, not the second. I wonder if this means that, instead of having a new `CreateCerts`, we could just enhance the `ManagedAliases` condition by:
```
ManagedAliases: !And 
- existing cond
- existing cond
- !Condition CreateALB
```

Reply:
Good call. We don't use `ManagedAliases` as well. `CustomDomainFunction` seems to be the only one but it's more like a missed one